scripts/update-workflows.js

```diff
@@ -35,14 +36,28 @@ function generateWorkflow({
 	workflow.jobs.test.with.package = pkg;
 	workflow.jobs.test.with['stylelint-version'] = stylelintVersion;
 
-	if (config['test-command']) {
-		workflow.jobs.test.with['test-command'] = config['test-command'];
+	if (config['set-overrides-command']) {
```

Comment:
minor

`yarn` supports `resolutions`.
i.e. the command itself could be overridden